config/jetson-node-k2-rear.json

```diff
@@ -0,0 +1,95 @@
+{
+  "version": 2,
+  "robot": {
+    "modules": {
+      "camera": {
+          "driver": "http",
+            "in": [],
+            "out": ["raw"],
+            "init": {
+              "url": "http://192.168.0.32/image?res=half",
+              "sleep": 0.1,
+              "timeout": 1.0
+          }
+      },
+      "camera_back": {
+          "driver": "http",
+            "in": [],
+            "out": ["raw"],
+            "init": {
+              "url": "http://192.168.0.33/image?res=half",
+              "sleep": 0.1,
+              "timeout": 1.0
+          }
+      },
+      "detector":{
+        "driver": "subt.jetson_artf_node:ArtifactDetectorJetson",
+        "in": ["image"],
+        "out": ["localized_artf", "dropped", "debug_image", "debug_result"],
+        "init": {
+          "camera_pose": [[0.5, 0, 0.375], [0, 0, 0, 1]]
+        }
+      },
+      "detector_rear":{
+        "driver": "subt.jetson_artf_node:ArtifactDetectorJetson",
+        "in": ["image"],
+        "out": ["localized_artf_rear", "dropped_rear", "debug_image_rear", "debug_result_rear"],
```

Comment:
`debug_image` ... why `rear`? It is already in the module name

Reply:
There are both front and rear cameras